images/Dockerfile

```diff
@@ -24,11 +24,26 @@ RUN export DOCKER_ARCH=x86_64 \
 
 FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
 
-ENV RUNNER_ALLOW_RUNASROOT=1
+ENV DEBIAN_FRONTEND=noninteractive
 ENV RUNNER_MANUALLY_TRAP_SIG=1
 ENV ACTIONS_RUNNER_PRINT_LOG_TO_STDOUT=1
 
-WORKDIR /actions-runner
-COPY --from=build /actions-runner .
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends \
+    sudo \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN adduser --disabled-password --gecos "" --uid 1001 runner \
+    && groupadd docker --gid 123 \
```

Comment:
Do we want to extract those as ARGs? Is there ever a scenario where we'll want to change those at build time?

Reply:
I will change it to ARG once we need to. 😄